www/kubl05/SP1/css/styles.css

```diff
@@ -139,6 +139,7 @@ footer {
 
 .title {
     font-size: 35px;
+    width: 204.917px;
```

Comment:
![image](https://user-images.githubusercontent.com/20724910/47850828-e6387680-ddd6-11e8-887b-47a76d3f3ed9.png)

tahle oprava nefunguje, u me to funguje kdyz dam pevnou sirku do jeho containeru, tj. headeru, resizuju to az na 386px a zpatky na 950 px